### ISSUE TITLE
[ide] field completion with keyworded field name part entered fails

### ISSUE THREAD

**Issue by nadako** [HaxeFoundation/haxe#10121]
```haxe
function main() {
	var o = {staticField: 3};
	o.static|
}
```

Completion here seems to yield toplevel completion instead of suggesting `staticField`

**Comment by Simn** [comment]
Yeah, hmm... I think that's a limitation we'll have to live with. This should be somewhat rare because you usually don't suddenly request completion after partially typing in a field name that just so happens to spell a keyword...
